src/views/Events/Events.module.scss

```diff
@@ -0,0 +1,26 @@
+.searchPageTitle {
+  color: var(--mui-palette-secondary-main);
+}
+.event_filter_icon {
+  color: var(--mui-palette-neutral-500);
+  font-size: 20;
+}
+.event_filter_list_icon {
+  color: var(--mui-palette-neutral-500);
+  font-size: 20;
+}
+.eventIcon {
+  color: var(--mui-palette-neutral-500);
+  font-size: 20;
+}
+.attended_clw {
+  background-color: var(--mui-palette-secondary-main);
+  color: var(--mui-palette-secondary-contrastText);
+}
+.filters {
+  background-color: var(--mui-palette-secondary-main);
+  color: var(--mui-palette-secondary-contrastText);
+}
```

Comment:
The `Chip` component has the standard `color` prop, so please use that instead.

Reply:
Fair enough, changing to the color prop did change the color of the "x" button to be white instead of gray for some reason, but I think it looks fine still.